src/components/Icon.tsx

```diff
@@ -250,6 +256,18 @@ export const Icons = {
   arrowBack: (
     <path d="M20.9999 11H6.41394L11.7069 5.70697L10.2929 4.29297L2.58594 12L10.2929 19.707L11.7069 18.293L6.41394 13H20.9999V11Z" />
   ),
+  arrowLeft: (
+    <path d="M20.9999 11H6.41394L11.7069 5.70697L10.2929 4.29297L2.58594 12L10.2929 19.707L11.7069 18.293L6.41394 13H20.9999V11Z" />
+  ),
+  arrowUp: (
+    <path d="M8.793 19.207L8.793 4.62097L14.086 9.91397L15.5 8.49997L7.793 0.792968L0.0859992 8.49997L1.5 9.91397L6.793 4.62097L6.793 19.207L8.793 19.207Z" />
+  ),
+  arrowDown: (
+    <path d="M6.79294 0.792909L6.79294 15.3789L1.49994 10.0859L0.0859372 11.4999L7.79294 19.2069L15.4999 11.4999L14.0859 10.0859L8.79294 15.3789L8.79294 0.792909L6.79294 0.792909Z" />
+  ),
+  arrowRight: (
+    <path d="M0.586001 8.99991L15.172 8.99991L9.879 14.2929L11.293 15.7069L19 7.99991L11.293 0.292908L9.879 1.70691L15.172 6.99991L0.586001 6.99991L0.586001 8.99991Z" />
+  ),
```

Comment:
Can we update these arrows to make sure they are centered in the 24x24 bounding box? (cc @Laurenmwoods)
See:
![Screen Shot 2022-08-11 at 12 12 53 PM](https://user-images.githubusercontent.com/1143861/184180310-f087d599-a0da-4d96-afc7-6a60785a90f0.png)

Reply:
Just took a look at the Figma, that looks good. @kbesingeryeh, I think you'll just need to make sure when you export the SVG from figma that you are selecting the 24x24 bounding box, and not just the icon

![Screen Shot 2022-08-11 at 12 17 52 PM](https://user-images.githubusercontent.com/1143861/184181061-991f92c7-d42b-4269-b10a-085afd0d4206.png)
.